Annotation/constituent_golden_set.json

```diff
@@ -1048,7 +1184,7 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "what bus to catch to go where you wish",

```

Comment:
I don't think it's a free relative because you can replace "what" with "which" (and "which" clauses are never free relatives according to our notes)